openmls/src/tree/tests_and_kats/unit_tests/test_resolution.rs

```diff
@@ -178,3 +182,61 @@ fn test_original_child_resolution() {
         );
     }
 }
+
+/// Test the `original_child_resolution` function that is used to calculate
+/// parent hashes
+#[test]
+fn test_exclusion_for_parent_nodes() {
+    // Create a large tree members.
+    let managed_group_config = ManagedGroupConfig::test_default();
+    let number_of_clients = 16;
+    let setup = ManagedTestSetup::new(managed_group_config.clone(), number_of_clients);
+
+    let group_id = setup.create_group(Ciphersuite::default()).unwrap();
+
+    let mut groups = setup.groups.borrow_mut();
+    let group = groups.get_mut(&group_id).unwrap();
+
+    let (_, group_creator_id) = group.members.first().unwrap().clone();
+
+    // We add 62 members so that we have a group of 63.
```

Comment:
How do we get to 62? The number of clients is 16 in line 192.

Reply:
Uh, leftover from before I realized I could reproduce the bug with a smaller group. I'll fix.